tests/js/export/default-escaped/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`default-escaped.js format 1`] = `
+====================================options=====================================
+parsers: ["babel"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+export asyn\\u{63} from "async";
+
+=====================================output=====================================
+export async from "async";
```

Comment:
Maybe we should start consider keep those `Identifier`s as it is.

Reply:
**Prettier pr-8914**
[Playground link](https://5f2a633bab50110008bac54e--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACAHQIFdgAmABgF88AzbCAW0JAEMBnATyjAJAG4QAGhAQMMAJbR2yUK2wMA7gAV5CGSlYAbRa04yRAI2yswAazgwAyhlMSoAc2QxsxOCLiNDcACY-fADKsjsSsDnAAYjiMrDCSjshsxDAQwiAAFjCMWgDq6RLw7LZgcFbqBRIAbgWciWDsBiD27HC4yiYOMci02i0iAFbsaABCJuaWVqyMcAH2cN297iCDaFb2DlpwAIrEEPALWn0gttgt2ImGrN5aaRjY9jA5Ej4w6cgAHBQidxAtOSYYRJ3OBnSrzEQAR128HaYg0bHYAFooHBfL40tg4FCJJj2mEukgeocli1GBJnK4SetNjs9vNCYsRDArk8Xm8kGQmSYJFp1gBhJgE1DsACsaWILQAKlcNESjpU3ABJKD+WBWMD3cQAQRVVhgnE2BxaVCoQA)
```sh
--parser babel
```

**Input:**
```jsx
export \u{20} from "async";
```

**Output:**
```jsx
export   from "async";

```

**Second Output:**
```jsx
SyntaxError: Unexpected token, expected "{" (1:10)
> 1 | export   from "async";
    |          ^
  2 | 
```

Reply:
**Prettier pr-8914**
[Playground link](https://5f2a633bab50110008bac54e--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACAHQIFdgBmABioF88AzbCAW0JAEMBnATyjAJADcIADQgIGGAEtoHZKDbZGAdwAKChLJRsANkrZdZogEbY2YANZwYAZQxnJUAObIY2YnFFwmRuABNffgAybE7EbI5wAGI4TGwwUk7I7MQwECIgABYwTNoA6hmS8Bx2YHDWGoWSAG6FXElgHIYgDhxwuCqmjrHIdDqtogBWHGgAQqYWVtZsTHCBDnA9fR4gQ2jWDo7acACKxBDwi9r9IHbYrdhJRmw+2ukY2A4wuZK+MBnIABwUovcQrbmmDBJe5wc5VBaiACOe3gHXEmnYHAAtFA4H4-OlsHBoZIsR1wt0kL0jstWkxJC43KSNltdvsFkSlqIYNdnq93kgAEzM0ySbQbADCzEJqA4AFZ0sRWgAVa6aYnHKruACSUACsGsYAeEgAgmrrDAuFtDq1qNQgA)
```sh
--parser babel
```

**Input:**
```jsx
export \u{3000} from "async";
```

**Output:**
```jsx
export 　 from "async";

```

**Second Output:**
```jsx
SyntaxError: Unexpected token, expected "{" (1:10)
> 1 | export 　 from "async";
    |          ^
  2 | 
```

Reply:
Turns out we can't test 

```js
export asyn\\u{63} from "async";
```

We should raise an issue after this merge.

Reply:
> We should raise an issue after this merge.

Agree

Reply:
#8917